nvdaHelper/interfaces/nvdaInProcUtils/nvdaInProcUtils.idl

```diff
@@ -74,6 +74,22 @@ interface NvdaInProcUtils {
 
 	error_status_t IA2Text_findContentDescendant([in] handle_t bindingHandle, [in] const unsigned long hwnd, [in] long parentID, [in] long what, [out] long* descendantID, [out] long* descendantOffset);
 
+
+	cpp_quote("/* Actually maps to nvdaInProcUtils_getTextFromIAccessible")
+	cpp_quote("*/")
+	error_status_t getTextFromIAccessible(
+		[in] handle_t bindingHandle,
+		[in] const unsigned long hwnd,
+		[in] long parentID,
+		// Params for getTextFromIAccessible
+		[out, string] BSTR* textBuf,
+		[in, defaultvalue(FALSE)] const boolean useNewText,

```

Comment:
This argument should be removed as it cannot be used cross-process. IAccessibleText::newText is only valid during an in-process winEvent callback. E.g. IA2_EVENT_TEXT_INSERTED. Out-of-process winEvent callbacks are asynchronous and therefore newText will no longer contain the correct information.

Reply:
I'll remove this from the params now and document it. Then in #13127 split this function into multiple responsibilities.